int32     move_speed
float32[] pose
---
bool success